### PR TITLE
Feat/create embedded component

### PR DESCRIPTION
**Pull request by Immo-Be**
also complets #1349 

- update chart with correct data
- allow description text with embedded content

image

### CODE REVIEW

File: src/scripts/components/stories/story/story.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ import {GalleryItemType} from '../../../types/gallery-item';
 import {useThunkDispatch} from '../../../hooks/use-thunk-dispatch';
 
 import styles from './story.module.styl';
+import StoryEmbedded from '../story-embedded/story-embedded';
```
Comment (andreashelms) [inline src/scripts/components/stories/story/story.tsx:30]:
```suggestion
import StoryEmbedded from '../story-embedded/story-embedded';

import styles from './story.module.styl';
```

File: src/scripts/types/gallery-item.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export interface VideoItem {
 
 export interface EmbeddedItem {
   type: GalleryItemType.Embedded;
+  text?: string;
```
Comment (andreashelms) [inline src/scripts/types/gallery-item.ts:29]: Maybe better `description` 🤔

Comment (Immo-Be) [reply src/scripts/types/gallery-item.ts:29]: It was `text` before, that's we I kept it. But might be a good opportunity to use something more descriptive :)

File: src/scripts/components/stories/story/story.tsx (hunk 1 above)
Comment (Immo-Be) [reply src/scripts/components/stories/story/story.tsx:30]: 👍

File: src/scripts/types/gallery-item.ts (hunk 2 above)
Comment (andreashelms) [reply src/scripts/types/gallery-item.ts:29]: You are right, for other gallery types it's also text. So let's just keep that.

**Review by andreashelms — APPROVED** [review]

**Pull request merged**